README.md

```diff
@@ -14,8 +14,8 @@ An Automatic Page Generation Tool http://jxnblk.com/doctor-mark
   - Meta description
   - Meta author
   - Meta keywords
-  - Github link
-  - NPM link
+  - GitHub link
+  - npm link
```

Comment:
I'm using standard English capitalization for proper nouns here. While I get that some people don't like this, I'm not really convinced to change it. That said `NPM` should probably be `Npm` since it's not an abbreviation.